pkg/shp/resource/resource.go

```diff
@@ -0,0 +1,80 @@
+package resource
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/shipwright-io/cli/pkg/shp/params"
+	"github.com/shipwright-io/cli/pkg/shp/util"
+)
+
+type ShpResource struct {
+	gv       schema.GroupVersion
+	kind     string
+	resource string
+
+	params *params.ShipwrightParams
+
+	resourceInterface dynamic.ResourceInterface
+}
+
+func (sr *ShpResource) getResourceInterface() (dynamic.ResourceInterface, error) {
```

Comment:
As a general comment, usually we have a single letter "this" reference. Could we keep this standard here, in this project, as well?

Reply:
Will do